syft/serde/protobuf/serde.py

```diff
@@ -215,8 +219,13 @@ def serialize(
 
     protobuf_obj = _bufferize(worker, obj)
 
-    if type(obj) == type(None):
+    obj_type = type(obj)
```

Comment:
Sorry, I've missed this in the beginning, but why do we need to fit everything into SyftMessagePB?
This limits `serialize` to types allowed in SyftMessagePB

Reply:
In order to properly deserialize a Protobuf message, you need to know the type. Since we don’t know which message to expect, the `SyftMessage` wrapper type makes the message type always the same, which makes deserialization easy. If we didn’t have it, we could try to parse an incoming message as each type in sequence and use whichever worked, but that would be kinda messy.